hypothesis-python/tests/redis/test_redis_exampledatabase.py

```diff
@@ -142,3 +142,8 @@ def listener(event):
     db.save(b"a", b"c")
     flush_messages(db)
     assert calls == 3
+
+
+def test_equality():
+    # FakeRedis() != FakeRedis(), not much we can do here
+    assert RedisExampleDatabase(FakeRedis()) != RedisExampleDatabase(FakeRedis())
```

Comment:
ugh fine for now but let's open a fakeredis issue?

Reply:
(again, forgot about the `is` case - added a check for `assert RedisExampleDatabase(redis := FakeRedis()) == RedisExampleDatabase(redis)`)